### PR TITLE
Ckeyless humans no longer get targeted by the assasinate objective

### LINKED ISSUE #5663: Assasinate objective covers SNPCs
**Problem Description**:
The system can assign SNPCs from the gateway as assasinate targets.

**What did you expect to happen**: 
SNPCs to not be taken in consideration.

**What happened instead**:
Traitor got an SNPC as an objective to assassinate.

**Why is this bad/What are the consequences:**
It's not a crew member and a player to be considered for assassination objectives.

**Steps to reproduce the problem**:
The system must pick at random a SNPC while they are at availible at round start.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: No.

### PR DESCRIPTION
**Pull request by FlattestGuitar**
:cl: FlattestGuitar
fix: SNPCs no longer get targeted by assasinate objectives
/:cl:

Fixes #5663

### CODE REVIEW

File: code/game/gamemodes/objective.dm [hunk 1]
```diff
@@ -35,6 +35,8 @@ var/list/potential_theft_objectives = subtypesof(/datum/theft_objective) \
 		return TARGET_INVALID_NOT_HUMAN
 	if(!possible_target.current.stat == DEAD)
 		return TARGET_INVALID_DEAD
+	if(possible_target.key == null)
```
Comment (Crazylemon64) [inline code/game/gamemodes/objective.dm]: Wouldn't `mind` be a better check for being a player-controlled character?

Comment (FlattestGuitar) [inline code/game/gamemodes/objective.dm]: `possible_target` IS the mind. It's either this or `possible_target.current--`further magic?

Comment (Crazylemon64) [inline code/game/gamemodes/objective.dm]: OH! oops, alright, this makes sense then

Comment (tigercat2000) [inline code/game/gamemodes/objective.dm]: don't do `== null` please, just do `if(!possible_target.key)`

Comment (Fox-McCloud) [inline code/game/gamemodes/objective.dm]: Yup, what Tiger said.

**Pull request merged**